### ISSUE TITLE
How to merge project to this repo?

### ISSUE THREAD

**Issue by TomTheBotter** [j-e-k/pgoapi#56]
I've added new features and fixed several issues for the next update. How can I merge my project to yours? PS I sent you an email

**Comment by wesselvdv** [comment]
Fork this repository, commit your changes and submit a pull request. I would presume that we can all see the issues you fixed, and updates you've made in relation to the original code.

**Comment by j-e-k** [comment]
Saw your email, we might as well communicate in public to facilitate open development/other's comments. 

In order to prevent a License Violation/DMCA takedown, the first (and really only) thing you would need to do is re-add the License to your repository. 

If you would like, attributing me in your README or re-creating your project as a fork of your repo and merging your changes in through a pull request would be appreciated. However, neither of these is strictly required by the License. 

Not so long story short, please re add the licenses to your project. Then if you want to work together in the spirit of open source, fork/open a pull request and lead users to this repository so support and development can be consolidated.

**Comment by j-e-k** [comment]
Paging @TomTheBotter

**Comment by beeedy** [comment]
I would like to note that there are a portion of changes in his repo that are not his work, but other peoples (mine included). If we could fine a way to give credit where due that would be much appreciated!

**Comment by wesselvdv** [comment]
@beeedy I would expect nothing less of other people. I would be disappointed if credit wouldn't be given to those who deserve it.

**Comment by TomTheBotter** [comment]
Yes I'll be adding the names of all people who contributed to the license and modifications copyright by today

**Comment by j-e-k** [comment]
@TomTheBotter Do you want to just make your project a fork of this repo and merge your changes back in?

**Comment by sontek** [comment]
@TomTheBotter Are you working on getting your changes merged in here? I noticed you disabled issues and PRs on yours (and yours currently doesn't work with latest master)

**Comment by beeedy** [comment]
@sontek He was removing all posts that mentioned this project as the original until he just disabled issues and pr's in general

**Comment by j-e-k** [comment]
😢
